index.js

```diff
@@ -3,9 +3,14 @@ loggerInit();
 log.time("startup");
 log.info("Starting Cutiebot!");
 
+if (process.env.DISCORD_TOKEN === undefined) {
+	log.error("Token not provided. Set the DISCORD_TOKEN environment variable and restart.");
+	process.exit(1);
+}
+
```

Comment:
This might fit better after the token variable declaration before client is created in terms of readability we then have all the constants at the top of the file

Reply:
Yes, you're right. It looks a lot better like that too!

```js
const token = process.env.DISCORD_TOKEN;

if (!token) {
  log.error("Token not provided. Set the DISCORD_TOKEN environment variable and restart.");
  process.exit(1);
}
```